src/ts/proxyfowarder.ts

```diff
@@ -0,0 +1,159 @@
+import { keccak256, defaultAbiCoder, solidityKeccak256 } from "ethers/utils";
+import { Wallet } from "ethers/wallet";
+import { ReplayProtectionAuthority } from "./replayprotectionauthority";
+import { ChainID, ProxyAccountDeployer, ProxyAccountFactory } from "..";
+import { ForwardParams, Forwarder, ProxyCallData } from "./forwarder";
+
+/**
+ * A single library for approving meta-transactions and its associated
+ * replay protection. All meta-transactions are sent via proxy contracts.
+ */
+export class ProxyForwarder extends Forwarder<ProxyCallData> {
```

Comment:
Name should be ProxyAccountForwarder

Reply:
changed

Reply:
There are still many references to proxyfactory

Reply:
there shouldnt be know